pipelines/rj_crm__disparo_template/utils/dispatch.py

```
@@ -951,22 +980,24 @@ def get_failed_cpfs(billing_project_id: str, campaign_name: str,) -> set:
     """
+    campaign_filter = f"AND LOWER(nome_hsm) = LOWER('{campaign_name}')" if campaign_name else ""
     query = f"""
         -- Agora só teremos Salesforce como fonte
         SELECT DISTINCT cpf
         FROM `rj-crm-registry.brutos_salesforce.status_disparo`
         WHERE indicador_quarentena = TRUE
         AND envio_datahora >= datetime_sub(current_datetime("America/Sao_Paulo"), INTERVAL 4 hour)
         AND data_particao >= date_sub(current_date("America/Sao_Paulo"), INTERVAL 1 day)
-        AND LOWER(nome_hsm) = LOWER('{campaign_name}') if campaign_name else ''
+        {campaign_filter}
     """
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)